lib/rdoc/generator/template/sdoc/resources/panel/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>search index</title>
+  <title>Search Index</title>
```

Comment:
Some of the titles looks inconsistently capitalised, so I also updated the ones I noticed as they're super close to the head/html tags.

Reply:
Yes, that makes sense. Although I'm not sure the search index is used anymore with the non-frames layout.